custom/conf/app.example.ini

```diff
@@ -2605,3 +2605,7 @@ LEVEL = Info
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; storage type
 ;STORAGE_TYPE = local
+
+;[user]
```

Comment:
The section name "user" reads a little strange (just a nit, not blocker)

Another question is: will there be more config options in this section (any plan) ? If no, if this section only has one option .... it seems to be somewhat over-engendering. (also a nit and a question).

Reply:
Yes, there will be more like password, rename, 2fa, etc. ref https://github.com/go-gitea/gitea/pull/20549 These came from real requirements.

Reply:
I couldn't find a better section name than `user`. Or maybe `user_setting`.

Reply:
No more currently but I can't find a better section.

Reply:
Changed in 9e928cacba8bf04df47c06bf13147457e3f43a78